stdlib/grpc/src/main/ballerina/src/grpc/grpc_errors.bal

```diff
@@ -14,101 +14,92 @@
 // specific language governing permissions and limitations
 // under the License.
 
-# The details of a gRPC error.
-#
-# + message - Specific error message for the error
-# + cause - Cause of the error if this error occurred due to another error (probably from another module)
-public type Detail record {
-    string message;
-    error cause?;
-};
-
-# Represents the reason of the cancelled error.
-public const CANCELLED_ERROR = "{ballerina/grpc}CancelledError";
+# Represents the type of the cancelled error.
+public const CANCELLED_ERROR = "CancelledError";
```

Comment:
Do we still need to maintain constants values for the error? or shall we remove those

Reply:
No need these constants for the error but these used to create the ErrorType(https://github.com/ballerina-platform/ballerina-lang/blob/master/stdlib/grpc/src/main/ballerina/src/grpc/grpc_errors.bal#L116). Therefore, I keep this.

Reply:
In retryConfig, instead of ErrorType const, is it possible to pass the actual error type

Reply:
yes, we can use it. But it will affect the API. So, I skipped this. 
Can we do the changes in the API?

Reply:
Fixed in #26e0f03